### ISSUE TITLE
Error en space

### ISSUE THREAD

**Issue by FroggyTuber** [ZhengLinLei/stackblock.io#9]
Se buggea cuando mantienes la tecla space pulsada

**Comment by ZhengLinLei** [comment]
Description: When you keep the space pressing (spamming), the game spams the `fncStart()` multiple times.

Error in:
```js
    // PC Version
    // Run fncStart function when the user press space key
    window.addEventListener('keydown', function(e){
        e.preventDefault();

        if(e.keyCode === 32 || e.keyCode === 40){
            fncStart();
        }
    });
```


Video:

https://github.com/ZhengLinLei/stackblock.io/assets/77938581/69919553-c261-45db-88e0-e83fdaed0ebd

**Comment by ZhengLinLei** [comment]
Solution:
```js
     let keyFree = true; 
    // PC Version
    // Run fncStart function when the user press space key
    window.addEventListener('keydown', function(e){

        if(e.keyCode === 32 || e.keyCode === 40){
            e.preventDefault();
            console.log("space");
            if(keyFree) {
                console.log("libre");
                keyFree = false;
                fncStart();
            }
        }
    });

    window.addEventListener('keyup', function(e){

        if(e.keyCode === 32 || e.keyCode === 40){
             keyFree = true;
             console.log("liberado");
        }
    });
```

**Comment by ZhengLinLei** [comment]
Fixed [e210c26](https://github.com/ZhengLinLei/stackblock.io/commit/e210c26d1894633e07538ec9d890fa6aaa472adf)
